ee/models/license.py

```diff
@@ -58,6 +58,7 @@ class License(models.Model):
         "organizations_projects",
         "google_login",
         "dashboard_collaboration",
+        "event_property_collaboration",
```

Comment:
I think this could be something a bit more generic like `ingestion_taxonomy` or `event_taxonomy`, we would probably use the same attribute for action taxonomy too.